xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -220,22 +220,24 @@ void CAESinkPipewire::EnumerateDevicesEx(AEDeviceInfoList& list, bool force)
   auto loop = pipewire->GetThreadLoop();
   loop->Lock();
 
-  CAEDeviceInfo device;
-  device.m_deviceType = AE_DEVTYPE_PCM;
-  device.m_deviceName = "Default";
-  device.m_displayName = "Default";
-  device.m_displayNameExtra = "Default Output Device (PIPEWIRE)";
-  device.m_wantsIECPassthrough = true;
+  CAEDeviceInfo defaultDevice;
```

Comment:
use brace initialization / initlist ?

Reply:
do you mean
```
  CAEDeviceInfo defaultDevice = {};
```
or
```
  CAEDeviceInfo defaultDevice = {
    .m_deviceName = "Default",
    .m_displayName = "Default",
    .m_displayNameExtra = "Default Output Device (PIPEWIRE)",
    .m_deviceType = AE_DEVTYPE_PCM,
    .m_wantsIECPassthrough = true,
  };
```

Reply:
I mean the latter except the named init is c++20 so it would be without the names.

Reply:
I don't really see a benefit as it decreases readability and requires mixed ways of setting members anyway.
```
 CAEDeviceInfo defaultDevice = {
    "Default",
    "Default",
    "Default Output Device (PIPEWIRE)",
    AE_DEVTYPE_PCM,
  };

  defaultDevice.m_wantsIECPassthrough = true;

```

Reply:
Benefit is no need to call default ctors for strings followed by assignment. But never mind, it isnt perf critical code.